user/firejail/template.py

```diff
@@ -0,0 +1,23 @@
+pkgname = "firejail"
+pkgver = "0.9.72_git20250221"
+pkgrel = 0
+_commit = "733f9a9c487358ba73b06186ed348f086d8ac839"
+build_style = "gnu_configure"
+configure_args = ["--sysconfdir=/usr/share"]
```

Comment:
this looks wrong

Reply:
By default, the profiles are put in /etc/profiles, but I understood that the package manager should not own files in /etc/. Please advice on where to put them (/usr/lib/firejail/profiles?).

Reply:
There seems to be no simple and elegant way to put the profiles in another directory than /etc/firejail (and thus avoiding that the  package owns files in /etc).